src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/SetOperation.java

```diff
@@ -73,14 +75,21 @@ public boolean shouldApplyIf(boolean isInsert) {
   @Override
   public boolean updateDocument(ObjectNode doc) {
     boolean modified = false;
+    Set<String> setPaths = new HashSet<>();
```

Comment:
Is this used anywhere? Looks like it isn't?

Reply:
Yes, in the new condition to check if `$vectorize` update clause is available

Reply:
Ah. I somehow missed that. Thanks!